README.md

```diff
@@ -1,5 +1,7 @@
 # cisco-cookbook
 
+[![Build Status](https://travis-ci.org/cisco/cisco-network-chef-cookbook.svg?branch=develop)](https://travis-ci.org/cisco/cisco-network-chef-cookbook)
+
```

Comment:
What about cookbook version info.. similar to what we have for the node_utils gem?